pytests/test_pyhelios.py

```diff
@@ -35,23 +35,58 @@ def test_sim():
     """
     Fixture which returns a simulation object for a given survey path
     """
-    def create_test_sim(survey_path):
+    def create_test_sim(survey_path, zip_output=True, las_output=True):
         # pyhelios.loggingSilent()
         from pyhelios import SimulationBuilder
         simB = SimulationBuilder(
             surveyPath=str(survey_path.absolute()),
             assetsDir=WORKING_DIR + os.sep + 'assets' + os.sep,
             outputDir=WORKING_DIR + os.sep + 'output' + os.sep,
         )
-        simB.setLasOutput(True)
+        simB.setLasOutput(las_output)
         simB.setRebuildScene(True)
-        simB.setZipOutput(True)
+        simB.setZipOutput(zip_output)
 
         sim = simB.build()
 
         return sim
 
     return create_test_sim
+    
+    
+def test_open_output(test_sim):
+    """Test accessing output files (LAS/xyz) after simulation"""
+    from pyhelios import SimulationBuilder
+    
+    def access_output(outpath, outfile_ending):
+        # Attempt to rename files in output directory
+        for filename in os.listdir(os.path.dirname(outpath)):
+            if filename.endswith(outfile_ending):
+                os.rename(os.path.join(os.path.dirname(outpath), filename), os.path.join(os.path.dirname(outpath), filename.replace(outfile_ending, "_accessed"+outfile_ending)))
```

Comment:
Shall we move that function outside the test?

Instead of renaming the file, can we also check accessibility is one of the ways described [here](https://www.novixys.com/blog/python-check-file-can-read-write/) for example.

Reply:
I moved the function outside the test.
I think renaming the files is a good solution. It works just as well as the ways described in the link and allows an easy retrospective check to see for which file(s) accessing worked and for which file it failed.